packages/vm/src/context.rs

```diff
@@ -329,7 +324,7 @@ pub(crate) fn with_iterator_from_context<'a, 'b, S, Q: 'b, F, T>(
 where
     S: Storage,
     Q: Querier,
-    F: FnOnce(&'b mut (dyn Iterator<Item = FfiResult<(KV, u64)>>)) -> VmResult<T>,
+    F: FnOnce(&'b mut Box<dyn StorageIterator + 'b>) -> VmResult<T>,
```

Comment:
I don't understand why I have to add this `Box` here. With the native `Iterator` it just worked

Reply:
addressed in dac1964

Reply:
I didn't actually do anything special in that commit. I think that the previous commit just sorted out the problem.

Reply:
Thanks!